lib/usecases/get_surveys_use_case.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter_survey/api/exception/network_exceptions.dart';
+import 'package:flutter_survey/api/repository/survey_repository.dart';
+import 'package:flutter_survey/model/survey_model.dart';
+import 'package:flutter_survey/usecases/base/base_use_case.dart';
+import 'package:injectable/injectable.dart';
+
+class GetSurveysInput {
+  final int pageNumber;
+  final int pageSize;
+
+  GetSurveysInput({
+    required this.pageNumber,
+    required this.pageSize,
+  });
+}
+
+@Injectable()
+class GetSurveysUseCase extends UseCase<List<SurveyModel>, GetSurveysInput> {
+  final SurveyRepository _surveyRepository;
+
+  const GetSurveysUseCase(this._surveyRepository);
+
+  @override
+  Future<Result<List<SurveyModel>>> call(GetSurveysInput input) {
```

Comment:
Here, why don't we use `SurveysModel` here? Should we use `List<SurveyModel>` for all and get rid of `SurveysModel`? 😕

Reply:
@luongvo Updated, I have removed `SurveysModel` 🚀